src/uu/fmt/src/linebreak.rs

```diff
@@ -371,11 +372,14 @@ fn build_best_path<'a>(paths: &[LineBreak<'a>], active: &[usize]) -> Vec<(&'a Wo
     let mut breakwords = vec![];
     // of the active paths, we select the one with the fewest demerits
     let mut best_idx = match active.iter().min_by_key(|&&a| paths[a].demerits) {
-        None => crash!(
-            1,
-            "Failed to find a k-p linebreak solution. This should never happen."
-        ),
         Some(&s) => s,
+        None => {
+            show!(USimpleError::new(
```

Comment:
Shouldn't we stop execution at this error? Either that, or if we can verify that it really never happens it should maybe be an `unreachable!` call.